.github/workflows/diktat.yml

```diff
@@ -10,14 +10,37 @@ jobs:
     runs-on: ubuntu-20.04
 
     steps:
-      - uses: actions/checkout@v3.0.0
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
       - name: Set up JDK 11
-        uses: actions/setup-java@v3.0.0
+        uses: actions/setup-java@v3
         with:
           java-version: 11
-          distribution: liberica
-      - name: Gradle Wrapper Validation
-        uses: gradle/wrapper-validation-action@v1.0.4
-      - uses: gradle/gradle-build-action@v2.1.5
+          distribution: zulu
+      - uses: gradle/gradle-build-action@v2
+        with:
+          gradle-version: wrapper
+          arguments: |
+            diktatCheck
+            -Pdiktat.githubActions=true
+            -Pdetekt.multiplatform.disabled=true
+      # Upload reports only if code style checks were failed
```

Comment:
I think it is OK to upload a SARIF report even when it's without errors.

Reply:
ok, np